cocos/editor-support/spine/SkeletonRenderer.cpp

```diff
@@ -98,7 +98,7 @@ SkeletonRenderer::~SkeletonRenderer () {
 	spSkeleton_dispose(_skeleton);
 	if (_atlas) spAtlas_dispose(_atlas);
 	if (_attachmentLoader) spAttachmentLoader_dispose(_attachmentLoader);
-	delete _worldVertices;
+	delete [] _worldVertices;
```

Comment:
The [pull request got merged](https://github.com/EsotericSoftware/spine-runtimes/pull/702) in the Spine runtime repository. We should be good to merge now.

Thanks a lot.